packages/sanity/src/structure/panes/document/statusBar/DocumentStatusBar.tsx

```diff
@@ -33,6 +44,22 @@ export function DocumentStatusBar(props: DocumentStatusBarProps) {
 
   const shouldRender = editState?.ready && typeof collapsed === 'boolean'
 
+  let actions: JSX.Element | null = null
+  if (createLinkMetadata && isCreateLinked(createLinkMetadata) && CreateLinkedActions) {
+    actions = (
+      <CreateLinkedActions
+        metadata={createLinkMetadata}
+        panelPortalElementId={DOCUMENT_PANEL_PORTAL_ELEMENT}
+        onDocumentChange={onDocumentChange}
+        documentTitle={title ?? ''}
```

Comment:
Is there a hook for getting default document title? Or is there a locale-resouce with "Untitled" I could use?